cli/dcoscli/package/main.py

```diff
@@ -450,10 +450,14 @@ def _search(json_, query):
     results = [index_entry.as_dict()
                for index_entry in package.search(query, config)]
 
-    emitting.publish_table(emitter,
-                           results,
-                           tables.package_search_table,
-                           json_)
+    if any(map(lambda element: (len(element.get('packages', [])) > 0),
```

Comment:
Can we use list comprehension instead of `map`? For this project we agreed to use list comprehension instead of `map`. E.g:
```python
if any([len(element.get('packages', [])) for element in results]):
  ...
```